autopilot-daemon/pkg/handlers/handler.go

```diff
@@ -144,6 +144,14 @@ func PingHandler() http.Handler {
 	return http.HandlerFunc(fn)
 }
 
+func InvasiveCheckHandler() http.Handler {
+	fn := func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("Launching invasive health checks. Results added to 'gpuhealth' node label"))
+		InvasiveCheckTimer()
```

Comment:
Maybe we should change this function name.. it seems like we're starting a timer, while instead we're just calling the test